Sources/SwiftDocC/Infrastructure/Link Resolution/LinkResolver.swift

```diff
@@ -63,7 +65,11 @@ public class LinkResolver {
         var renderReferenceDependencies: RenderReferenceDependencies
         /// The different source languages for which this page is available.
         var sourceLanguages: Set<SourceLanguage>
-        
+        /// The kind of external content that's being referenced.
+        ///
+        /// For example, the navigator requires specific knowledge about what type of external symbol is being linked to.
+        var documentationKind: DocumentationNode.Kind
```

Comment:
Is it necessary to store a `DocumentationNode.Kind` if we only need to access the symbol identifier that's derived from it?